anime_downloader/sites/animesvostfr.py

```diff
@@ -0,0 +1,247 @@
+from anime_downloader.sites.anime import Anime, AnimeEpisode, SearchResult
+from anime_downloader.sites import helpers
+import logging
+import re
+
+logger = logging.getLogger(__name__)
+
+class AnimesVOSTFR(Anime, sitename='animesvostfr'):
+       
+        sitename = 'animesvostfr'
+        QUALITIES = ['360p', '480p', '720p', '1080p']
+        DOMAIN = "https://www1.animesvostfr.net/"
+
+        @classmethod
+        def search(cls, query):
+            
+            soup = helpers.soupify(helpers.get(cls.DOMAIN, params={'s' : query}, cf=True))
+            
+            results = []
+            
+            for v in soup.select('div.movies-list > div.ml-item'):
+                t = v.h2.text
+                u = v.a['href']
+                p = v.img['src']
+                
+                lang = v.select_one('div.language').text.upper()
+                
+                #logger.debug( "results: {}".format(lang) )
+                
+                eps = v.select_one('span.mli-quality').text.upper()
+                
+                #logger.debug( "results: {}".format(eps) )
+                
+                if re.search('VOSTFR', lang, re.IGNORECASE):
```

Comment:
This could probably be done more beautifully with a dict + loop, but that's just my personal preference.

Reply:
OK i am Python beginner, so i can change it if you send me a better code

Reply:
No regex + easily expandable/changed
```
                lang_dict = {
                'VOSTFR':f' (VOSTFR {eps})',
                'VF':f' (VF {eps})',
                'VO':f' (VO {eps})'
                }
                selected_lang = f' (V? {eps})' #default lang
                for a in lang_dict:
                    if a in lang.upper():
                        selected_lang = lang_dict[a]
                        break
                t += selected_lang
```
should probably work, use it if you want, no requirements

Reply:
OK that is done, thks